featuretools/tests/primitive_tests/test_features_deserializer.py

```diff
@@ -275,10 +277,13 @@ def _check_schema_version(version, es, warning_text):
         'feature_definitions': {}
     }
 
-    if warning_text:
-        with pytest.warns(UserWarning) as record:
-            FeaturesDeserializer(dictionary)
-
-        assert record[0].message.args[0] == warning_text
+    # TODO Refactor this function and both parent functions to run tests with a less complex implementation
+    earlier_schema_warning = 'is no longer supported by this version'
+    if earlier_schema_warning in str(warning_text):
+        logger = logging.getLogger('featuretools')
+        logger.propagate = True
+        FeaturesDeserializer(dictionary)
+        assert warning_text in caplog.text
+        logger.propagate = False
```

Comment:
Similar to the entityset test, we need to make sure we are testing for the `UserWarning` for the later schema version test.